lib/ansible/modules/utilities/logic/import_playbook.py

```diff
@@ -29,6 +29,7 @@
       - The name of the imported playbook is specified directly without any other option.
 notes:
   - This is a core feature of Ansible, rather than a module, and cannot be overridden like a module.
+  - Note that including an empty playbook in a master playbook, will result in an error.
```

Comment:
It might make sense to give an example of error or smth

Reply:
you're right, I'm sorry.

the error generated is:
```
ERROR! playbooks must be a list of plays
```

Reply:
Updated patch.

Reply:
You don't have to apologize, you are doing great! It's just a typical iterative feedback/review-based improvement process :) Thanks for the patch!

Reply:
we might want to give a better error instead:
```

diff --git a/lib/ansible/playbook/__init__.py b/lib/ansible/playbook/__init__.py
index f365566f90..708e2f8563 100644
--- a/lib/ansible/playbook/__init__.py
+++ b/lib/ansible/playbook/__init__.py
@@ -79,7 +79,9 @@ class Playbook:
         except UnicodeDecodeError as e:
             raise AnsibleParserError("Could not read playbook (%s) due to encoding issues: %s" % (file_name, to_native(e)))

-        if not isinstance(ds, list):
+        if not ds:
+            raise AnsibleParserError("Empty playbook, nothing to parse", obj=ds)
+        elif not isinstance(ds, list):
             # restore the basedir in case this error is caught and handled
             self._loader.set_basedir(cur_basedir)
             raise AnsibleParserError("playbooks must be a list of plays", obj=ds)

```

Reply:
@bcoca +1 for improving the error message; that will go into 2.8, right?

Reply:
I'll see if i can backport, but probably wont make 2.7.0, but 2.7.1